HealthcareSystem/GraphicalEditor/Service/ServerService.cs

```diff
@@ -0,0 +1,119 @@
+﻿using GraphicalEditor.Enumerations;
+using GraphicalEditor.Models;
+using GraphicalEditor.Models.MapObjectRelated;
+using Model.Manager;
+using RestSharp;
+using System;
+using System.Collections.Generic;
+
+namespace GraphicalEditor.Service
```

Comment:
Methods in this class contains too many lines of code. Generally, any method longer than ten lines should make you start asking questions.

If you feel the need to comment on something inside a method, you should take this code and put it in a new method. Even a single line can and should be split off into a separate method, if it requires explanations. And if the method has a descriptive name, nobody will need to look at the code to see what it does.

To reduce the length of a method body, use Extract Method ( https://refactoring.guru/extract-method ).

Reply:
Yes, I agree. Refactoring has already started parallel to making a PR.

Reply:
Solved by splitting up the class in 227dd73d685d299759a4b4d3dbf857be00385e68 as far as I can see.